lib/commands/publish-draft.coffee

```diff
@@ -21,6 +21,9 @@ class PublishDraft
 
     @confirmPublish =>
       try
+        postassetfolder = path.join(path.dirname(@draftPath), utils.getTitleSlug(@draftPath))
+        fs.moveSync(postassetfolder,
+                    path.join(path.dirname(@postPath), @_getPostTitle())) if fs.existsSync(postassetfolder)
```

Comment:
Surely yes. Creating asset folder when image files are present is all that needs.
If we don't move asset folders respectively, users of this plugin will head for hexo commands undoubtedly ;-)